node/node.go

```diff
@@ -771,6 +771,10 @@ func (node *Node) StartMining(ctx context.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to get mining address")
 	}
+	_, err = node.PorcelainAPI.ActorGet(ctx, minerAddr)
+	if err != nil {
+		return errors.Wrap(err, "failed to get miner actor")
```

Comment:
A test for this should be added to `node_test.go` -- here is where I would start:
https://github.com/filecoin-project/go-filecoin/blob/199c9241887366995ea8493fc5d9136be9f09f3c/node/node_test.go#L142-L143

After the call to `GiveMiner` we should assert that calling `StartMining` returns the expected error, so something like this:
`	assert.Contains(t, minerNode.StartMining(ctx).Error(), "failed to get miner actor")`